File: AC/ABS/abc088/b.py
n = int(input())
a = 0
b = 0
arr = list(map(int, input().split()))
arr.sort(reverse=True)
for i in range(0, n):
    if i % 2 == 0:
        a += arr[i]
    else:
        b += arr[i]

print(a-b)

File: AC/ABS/abc088/b.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var n, asum, bsum int
	fmt.Scan(&n)
	arr := make([]int, n)
	for i := 0; i < n; i++ {
		fmt.Scan(&arr[i])
	}
	sort.Sort(sort.Reverse(sort.IntSlice(arr)))
	for i := 0; i < n; i++ {
		if i%2 == 0 {
			asum += arr[i]
		} else {
			bsum += arr[i]
		}
	}
	fmt.Println(asum - bsum)
}
